docs/reference/search/search-your-data/search-using-query-rules.asciidoc

```diff
@@ -0,0 +1,150 @@
+[[search-using-query-rules]]
+== Searching with query rules
+++++
+<titleabbrev>Searching with query rules</titleabbrev>
+++++
+
+[[query-rules]]
+
+preview::[]
+
+_Query rules_ are rules that are defined inside {es} that allow customization of search results for queries that match specified criteria metadata.
+This allows for more control over promoted results, ensuring that specified documents always match defined criteria and are returned at the top of the result list.
+Metadata is defined in the query rule, and is matched against the query criteria.
+Metadata is a string key, and can be anything from a user-entered query string, personalized metadata about users (e.g. country, language, etc). or anything else defined in the rules.
+
+Rules are defined using the <<query-rules-apis,Query rules API>> and searched using the <<query-dsl-rule-query,rule query>>.
+
+When defining a rule, consider the following:
+
+* The type of rule we want to apply:
++
+--
+* `pinned` will re-write the query into a <<query-dsl-pinned-query, pinned query>>, pinning specified results matching the query rule at the top of the returned result set.
+--
+* The criteria for which this rule will match. Criteria is defined as `type`, `metadata`, and `values`. The allowed types are:
++
+--
+* `exact` will match when the rule metadata matches the specified value exactly.
```

Comment:
Would a table be a better choice for having the type, description, and an example of match?

For example:
| prefix | matches when the rule metadata starts with the specified value | "pref" will match "prefix" |